client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -327,104 +327,33 @@
-                *ngIf="tags.length"
-                [disabled]="!selectedRows.length"
-                (click)="multiselectWrapper(multiselectService.changeTags(selectedRows))"
-            >
-                <mat-icon>bookmarks</mat-icon>
-                <!-- TODO: icon -->
-                <span translate>Add/remove tags</span>
-            </button>
-            <button
-                mat-menu-item
-                [disabled]="!selectedRows.length"
-                (click)="multiselectWrapper(multiselectService.moveToItem(selectedRows))"
-            >
-                <mat-icon>sort</mat-icon>
-                <span translate>Move to agenda item</span>
-            </button>
-            <button
-                mat-menu-item
-                [disabled]="!selectedRows.length"
-                (click)="multiselectWrapper(multiselectService.bulkMoveItems(selectedRows))"
-            >
-                <mat-icon>format_indent_increase</mat-icon>
-                <span translate>Move in call list</span>
-            </button>
-        </div>
-        <div *ngIf="perms.isAllowed('manage')">
-            <button mat-menu-item [disabled]="!selectedRows.length" (click)="openExportDialog()">
-                <mat-icon>archive</mat-icon>
-                <span translate>Export selected motions</span>
-            </button>
-            <mat-divider></mat-divider>
-            <button
-                mat-menu-item
-                class="red-warning-text"
-                [disabled]="!selectedRows.length"
-                (click)="multiselectWrapper(multiselectService.delete(selectedRows)); toggleMultiSelect()"
-            >
-                <mat-icon>delete</mat-icon>
-                <span translate>Delete</span>
-            </button>
+            <os-motion-menu-list [selectedMotions]="selectedRows" (action)="multiselectWrapper($event)">
+                <button
+                    mat-menu-item
+                    [disabled]="!selectedRows.length"
+                    (click)="multiselectWrapper(multiselectService.moveToItem(selectedRows))"
+                >
+                    <mat-icon>sort</mat-icon>
+                    <span translate>Move to agenda item</span>
+                </button>
+                <button
+                    mat-menu-item
+                    [disabled]="!selectedRows.length"
+                    (click)="multiselectWrapper(multiselectService.bulkMoveItems(selectedRows))"
+                >
+                    <mat-icon>format_indent_increase</mat-icon>
+                    <span translate>Move in call list</span>
+                </button>
+                <button
+                    *ngIf="perms.isAllowed('manage')"
+                    mat-menu-item
+                    [disabled]="!selectedRows.length"
+                    (click)="openExportDialog()"
+                >
+                    <mat-icon>archive</mat-icon>
+                    <span translate>Export selected motions</span>
```

Comment:
as above - Export selected motions for both the motion list and the amendment list